github/gh-collaboration.qmd

```diff
@@ -1,6 +1,13 @@
 # How to collaborate on Github
 
 ## Making a branch
+- On the main page of the repository go to the file tree view on the left and click on the branch dropdown menu.

```

Comment:
Sorry, what's the "file tree view"? I don't mean to be a pain on this, but following this, I'm not sure what you're referring to. The text should be unambiguous to a novice.

Reply:
Hmm, I took that exact formulation from here (the official documentation): 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository

I will make up my own novice word for it ;-), thanks for pointing it out.

Reply:
I guess the main difference is the use of pictures and icons there. We can try and enable the fontawesome icons if you think that will help.

Reply:
I think embedding their picture should be fine too as long as you attribute it.

Reply:
I have to admit though it feels a bit long-winded given the `branches` button to the right of it, that brings one to the same page in less clicks.

Reply:
I actually added one more sentence and the link. Hopefully it is clear now.